### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by markosjal** [funcy2267/kodi-script-launcher#2]
I tried to use this . I have an x86_64 install of libreELEC running but building on my Linux Mint x86_64 

I was able to build a script that should launch a simple python command calling it "myaddon"
On the first make install I found a .zip file in the addon directory script.myaddon on the Linux mint system and copied that to the LibreELEC system and installed it there from ZIP
After restarting libreELEC it did not start the script

so I figured remove it I would try again...

make uninstall did not seem to work on my linux mint, and I would not expect it to work on LibreELEC so the folder was removed on both.

So I went back thinking maybe I had to call it script.launcher,myscript and changed the name.
after make and make install there is no ZIP file this time but there is a new addon folder with the new name 

Turns out I do see a zip in the  build directory

How can I add dependencies to this? I need to ensure python and requests addons are installed.

is there a way to add files to the built zip or can I add them to the zip file after it is built?

any ideas on why my python script not running? Is the addon name I select important?

**Comment by funcyme** [comment]
Script you are launching is not python script, but bash script.
I also want to point out that I haven't tested this on LibreELEC, but only on native Linux installations.
About additional files in addon directory, maybe after some testing, where kodi opens everything I will add an option to do this.
The solution I recommend is to execute python scripts from bash with given path to this script. Then you only need to ensure dependencies are installed on system-side for python script to work properly.

**Comment by markosjal** [comment]
Yes but the BASH script launches a Python script. I am not looking to split hairs but will clarify that point

In this case they python launches from a BASH script because it uses a python wrapper due to limitations built into Libreelec

pythonwrapper.sh
`#!/bin/sh
PYTHONPATH=
for addon in /storage/.kodi/addons/*/lib /storage/.kodi/addons/*/libs /usr/lib/kodi/addons/*/lib; do
  [ -d "${addon}" ] && PYTHONPATH="${PYTHONPATH:+${PYTHONPATH}:}${addon}"
done
export PYTHONPATH
exec /usr/bin/python "$@"`

Then I launch the following to start it

`./pythonwrapper.sh .kodi/directory/myscript.py --PORT 8080 --PARAMETER1 "ABC" --PARAMETER2 "XYZ"`

I mean if it can run a BASH script why can it not run a bash script that launches python? I can launch the same script from autostart.sh in Libreelec or at the command prompt


And yes I use the full path to pythonwrapper when I tried it in the addon.


The other issue I face is ensuring the (python) "requests" addon is loaded as a dependency.

I thought this might be a useful way of sharing what I cobbled together in which it could be more easily installed/distributed but maybe not.
